manifests/mod/nss.pp

```diff
@@ -1,7 +1,8 @@
 class apache::mod::nss (
   $transfer_log = "${::apache::params::logroot}/access.log",
   $error_log    = "${::apache::params::logroot}/error.log",
-  $passwd_file  = undef
+  $passwd_file  = undef,
+  $nss_port     = 8443
```

Comment:
is there  a special reason to call this parameter nss_port, instead of just port?